tests/Action/ProcessorTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Jahuty\Action;
+
+use Jahuty\Api\Request;
+
+class ProcessorTest extends \PHPUnit\Framework\TestCase
+{
+    public function testProcessReturnsRequest(): void
+    {
+        $action = new Show('render', 1, ['bar' => 'baz']);
+
+        $this->assertInstanceOf(
```

Comment:
PHPUnit assert methods are static (recommended to call them with self instead of $this)

Reply:
Ha, I don't know why, but I prefer the `$this->assertTrue()`. If it helps, the PHPUnit docs say there is no wrong way to do it:

> A common question, especially from developers new to PHPUnit, is whether using $this->assertTrue() or self::assertTrue(), for instance, is “the right way” to invoke an assertion. The short answer is: there is no right way. And there is no wrong way, either. It is a matter of personal preference.